docs/UserGuide.md

```diff
@@ -21,251 +45,323 @@
 
 * Words in `lower_case` are the reserved keywords.
 
 * Items in `[SQUARE_BRACKETS]` are optional parameters.
 
-* Items in `<ANGLE_BRACKETS>` are partially optional parameters (a minimum
-  number of partial optional parameters are required for certain command).
+* Items in `<ANGLE_BRACKETS>` essential parameters. All essential parameters have to be included in the command.
 
 * Items separated by `|` are in parallel relation, only one of them should be
   use in each command.
 
 * Items with `...` after them can have multiple instances.
 
-### Viewing help : `help`
+* The parameters have to follow the order in which they are presented, unless otherwise specified.
 
-    help
+<a name="pk">**parameter keywords**</a>
 
-Help is also shown if you enter an incorrect command e.g. `abcd`
+|Keyword | Definition|
+|----| :--------:|:------|
+|`tdy` | today|
+|`tmr` | tommorow|
+|`yst` | yesteday|
+| `to` | to |
+|`jan` | January|
+|`feb` | February|
+|`mar` | March|
+|`apr` | April|
+|`may` | May|
+|`jun` | June|
+|`jul` | July|
+|`aug` | August|
+|`sep`| September|
+|`oct` | October|
+|`nov` | November|
+|`dec` | December|
 
-### Adding a floating task : `task`
+<a name="DateFormat">**Date Format**</a>
+Valid input examples:
 
-Adds a floating task to TaskTracker.
+ddmonthyy
 
-    task "FLOATING_TASK_NAME" [PRIORITY]
+	31dec2016
 
-* Task name should be in a pair of quotation marks. And quotations marks are
-not allowed in task name.
+ddmonth
 
-* Task name could be a single word or a phrase, white space are allowed.
+	31dec
+* If the year is not defined, the year is assumed to be the current year. The year this document was last updated was 2016, therefore all examples protraying the current year will read 2016.
 
-* The floating task will be shown according to their `PRIORITY`.
+dd
 
-* Two floating tasks with the same `PRIORITY` will be shown according to
-  the order of the time they created.
+	31
+* If the month is not defined, the month is assumed to be the current month.
 
-* The `PRIORITY` attribute is an integer which ranges from `0` to `5`.
 
-* The default priority of the floating task is 0.
+Today
 
-#### Examples
+	tdy
 
-* `task "EE2020 lab report" 5`
+* Means today, the current date as logged by the local machine.
 
-  To create a task called `EE2020 lab report` with `PRIORITY` of 5.
+Tommorow
 
-* `task ProgressReflection`
+	tmr
 
-  To create a task called `ProgressReflection` with default `PRIORITY` of 0.
+* Means tommorow, the day after today as logged by the local machine.
 
-### Adding a event-like task : `event`
+To
+	
+    to
+    
+ * This keyword is only applicable for adding an event. `to` can only be entered into an `<END_DATE>` field. `to` would mean the `<END_DATE>` is the same as the `<START_DATE>` for an event. See [example](#to).
 
-Adds an event with specific starting and ending date or time to TaskTracker.
+**<a name="TimeFormat">Time Format</a>**
+The 12 hour clock is used. hh:mm am/pm 
```

Comment:
So here it says we are using `hh:mm`, but below I see `830am` ????

By the way, we need to change all time formats to use `hh:mm` format e.g. `8:30am` due to limitations in Java's `DateTimeFormatter`.